mlnode/iotdb/mlnode/storage/model_storager.py

```diff
@@ -0,0 +1,113 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+import os
+import json
+import torch
+import shutil
+import torch.nn as nn
+from pylru import lrucache
+from iotdb.mlnode.constant import (MLNODE_MODEL_STORAGE_DIR,
+                                   MLNODE_MODEL_STORAGE_CACHESIZE)
+
+
+# TODO: Add permission check firstly
+# TODO: Consider concurrency, maybe
+class ModelStorager(object):
+    def __init__(self,
+                 root_path: str = 'ml_models',
+                 cache_size: int = 30):
+        current_path = os.getcwd()
+        self.root_path = os.path.join(current_path, root_path)
+        if not os.path.exists(self.root_path):
+            os.mkdir(self.root_path)
+        self._loaded_model_cache = lrucache(cache_size)
+
+    def save_model(self,
+                   model: nn.Module,
+                   model_config: dict,
+                   model_id: str,
+                   trial_id: str):
+        """
+        Return: True if successfully saved
+        """
+        fold_path = os.path.join(self.root_path, f'{model_id}')
+        if not os.path.exists(fold_path):
+            os.mkdir(fold_path)
+        sample_input = [torch.randn(1, model_config['input_len'], model_config['input_vars'])]
+        torch.jit.save(torch.jit.trace(model, sample_input),
+                       os.path.join(fold_path, f'{trial_id}.pt'),
+                       _extra_files={'model_config': json.dumps(model_config)})
+        return os.path.exists(os.path.join(fold_path, f'{trial_id}.pt'))
```

Comment:
We should use `try...except...`. If there is no exception, we can consider the model to be saved successfully.